debug.py

```diff
@@ -1,3 +1,10 @@
-import bash.code_to_gast.bash_main as bash_main
+import main as main
+from bootstrap import bootstrap
 
-print(bash_main.bash_to_gast('true'))
\ No newline at end of file
+# Register all languages before testing can begin
+bootstrap()
+
+print(
```

Comment:
Do we want to submit changes to debug.py?

Reply:
No - committed this accidentally